content/en/security/threats/workload_security_rules/_index.md

```diff
@@ -19,9 +19,68 @@ further_reading:
   text: "Learn more about Security notification variables"
 ---
 
-With Cloud Security Management Threats (CSM Threats) enabled, the Datadog Agent actively monitors system activity and evaluates it against a set of out-of-the-box rules to detect suspicious behavior. CSM Threats rules consist of two different components: [Agent rules](#agent-rules) and [detection rules](#detection-rules).
+This topic explains how Cloud Security Management Threats (CSM Threats) actively monitors system activity and evaluates it against a set of out-of-the-box (OOTB) rules to detect suspicious behavior.
 
-## Agent rules
+## CSM Threats rules construction
+
+CSM Threats rules consist of two different components: Agent rules and Threat detection rules.
+
+- **Agent rules:** [Agent rules][9] are evaluated on the Agent host. CSM Threats first evaluates activity within the Datadog Agent against Agent expressions to decide what activity to collect. Agent expressions use Datadog’s [Security Language (SECL)][2]. 
+  
+  For example, here is the *Agent Configuration rule* expression `cryptominer_args`: 
+
+  ```text
+  exec.args_flags in ["cpu-priority", "donate-level", ~"randomx-1gb-pages"] ||
+  exec.args in [
+      ~"*stratum+tcp*",
+      ~"*stratum+ssl*",
+      ~"*stratum1+tcp*",
+      ~"*stratum1+ssl*",
+      ~"*stratum2+tcp*",
+      ~"*stratum2+ssl*",
+      ~"*nicehash*",
+      ~"*yespower*"
+  ]
+  ```
+- **Threat Detection rules:** [Threat Detection rules][3] are evaluated on the Datadog cloud (backend). Threat Detection rules are composed of existing Agent Configuration rules and additional expression parameters.
```

Comment:
Suggesting that we use "Datadog" or "Datadog backend" instead of "Datadog cloud" which isn't a term that we use in the docs. I think my personal preference is for "Datadog", but for this PR, I went ahead and kept the use of "Datadog backend".